osgp/platform/osgp-adapter-domain-publiclighting/src/main/java/org/opensmartgridplatform/adapter/domain/publiclighting/infra/jms/OsgpCoreRequestMessageProcessor.java

```diff
@@ -28,22 +30,26 @@
     private AdHocManagementService adHocManagementService;
 
     public void processMessage(final RequestMessage requestMessage, final String messageType)
-            throws UnknownMessageTypeException {
+            throws UnknownMessageTypeException, NotImplementedException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.apache.commons.lang3.NotImplementedException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)